api/server.go

```diff
@@ -54,14 +55,14 @@ func (a *API) createLoadAgentHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	ltConfig := data.LoadTestConfig
-	if err := ltConfig.IsValid(); err != nil {
+	if err := defaults.Validate(ltConfig); err != nil {
 		writeResponse(w, http.StatusBadRequest, &Response{
 			Error: fmt.Sprintf("could not validate config: %s", err),
 		})
 		return
 	}
 
-	var ucConfig control.Config
+	var ucConfig interface{}
```

Comment:
Is it necessary for this to be `interface{}` ? The validate methods already takes `interface{}`.

Reply:
Actually this is not needed anymore. will remove it completely 👍

Reply:
Hmm .. this does not appear to be removed ?

Reply:
🤦 actually this is needed to use in `switch` statement.

Reply:
So then back to my original question: is it necessary for this to be `interface{}` ? :)